src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -33,9 +34,25 @@ public HearingManagementServiceImpl(final MessageSenderConfiguration messageSend
 
     @Override
     public void processRequest(String caseId, HearingDetailsRequest hearingDetailsRequest) {
-        cftHearingService.isValidCaseId(caseId);
+        Integer latestVersion = cftHearingService.getLatestVersion(caseId);
         isValidRequest(hearingDetailsRequest);
-        validateHmiHearingRequest(hearingDetailsRequest, caseId);
+        validateHmiHearingRequest(hearingDetailsRequest, caseId, latestVersion);
+    }
+
+    public boolean isValidRequestVersion(HearingDetailsRequest hearingDetailsRequest,
```

Comment:
```suggestion
    private void isValidRequestVersion(HearingDetailsRequest hearingDetailsRequest,
```
`private` as it's only being called internally; `void` because the `return` doesn't need to be used anywhere. Arguably the method name itself should be something closer to `validateRequestVersion()`, but I'll let you decide whether to change that too, as I guess you were just trying to be a bit consistent with what was there already in the similar method!

Reply:
Done